ansible/roles/vm_set/tasks/add_topo.yml

```diff
@@ -10,6 +10,11 @@
     net: none
     detach: True
     cap_add: NET_ADMIN
+    privileged: yes
+  become: yes
+
+- name: Enable ipv6 for docker container ptf_{{ vm_set_name }}
+  command: docker exec -i ptf_{{ vm_set_name }} sysctl -w net.ipv6.conf.all.disable_ipv6=0
```

Comment:
do we want to make it as a default setting in docker-ptf? then we could modify the docker-ptf Dockerfile.

Reply:
probably better to change this at runtime, we might want to enable disable at per interface level later.

Reply:
ok